### PR TITLE
Issue53

### PR DESCRIPTION
**Pull request by gagansk**
Provides a messaging mechanism to edit the Public URL of the Plugin.

### CODE REVIEW

**Comment by jbernal0019** [comment]
@gagansk 

I've upgraded the `package.json` to  chrisstoreapi@^1.1.8 .

 In the new client `client.modifyPlugin(pluginId, dockerImage, publicRepo)` changed to  `client.modifyPlugin(pluginId, publicRepo) ` as the `dockerImage ` is immutable. 

Also keep in mind that a plugin can be shared with another user (he/she becomes owner of the plugin too) by using this same method and passing the new owner as the last argument like `client.modifyPlugin(pluginId, publicRepo, newOwner) `  or `client.modifyPlugin(pluginId, undefined, newOwner) ` if you don't want to modify the `publicRepo` and just add another owner.

File: src/components/Dashboard/components/DashPluginCardView/DashPluginCardView.js [hunk 1]
```diff
@@ -61,42 +61,83 @@ class DashPluginCardView extends Component {
...
-    const primaryContent = <p className="lead">Are you sure?</p>;
-    const secondaryContent = (
+    const primaryDeleteContent = <p className="lead">Are you sure?</p>;
+    const secondaryDeleteContent = (
       <p>
         Plugin <b>{pluginToDelete ? pluginToDelete.name : null}</b> will be permanently deleted
       </p>);
+    const primaryEditContent = <p className="lead">Edit Plugin Details</p>;
+    const secondaryEditContent = (
+      <b>{pluginToEdit ?
```
Comment (jcaianirh) [inline src/components/Dashboard/components/DashPluginCardView/DashPluginCardView.js]: @gagansk can you use patternfly-react controls for editing the plugin

Comment (gagansk) [reply src/components/Dashboard/components/DashPluginCardView/DashPluginCardView.js]: @jcaianirh Form and FormGroup have replaced label and input. Thank you for pointing this out.

**Comment by gagansk** [comment]
Patternfly controls have been used in the latest commit.

**Comment by gagansk** [comment]
Used Grid and Col to make the changes asked for. 

image

File: src/components/Dashboard/components/DashPluginCardView/DashPluginCardView.js [hunk 2]
```diff
@@ -61,42 +67,91 @@ class DashPluginCardView extends Component {
...
+              </Col>
+              <Col sm={4}>
+                <FormControl
+                  type="text"
+                  defaultValue={pluginToEdit.public_repo}
+                  onChange={this.handlePublicRepo}
+                  name="publicRepo"
+                />
+                <HelpBlock>
+                  For people to see your code
```
Comment (jcaianirh) [inline src/components/Dashboard/components/DashPluginCardView/DashPluginCardView.js]: Enter the public repo URL for your plugin

**Comment by jcaianirh** [comment]
@gagansk can the textbox be a bit wider?

**Comment by gagansk** [comment]
@jcaianirh, does this look good?

image

**Pull request merged**